qml/Keyboard.qml

```diff
@@ -313,10 +356,45 @@ Item {
                 prevSwipePositionX = mouseX
                 prevSwipePositionY = mouseY
                 fullScreenItem.timerSwipe.stop()
+                
+                if (firstPress === Qt.point(0,0)) {
+                    firstPress = Qt.point(mouse.x, mouse.y)
+                }
             }
 
             onReleased: {
-                fullScreenItem.timerSwipe.restart()
+                if (!cursorSwipeArea.selectionMode) {
+                    fullScreenItem.timerSwipe.restart()
+                    firstPress = Qt.point(0,0)
+                } else {
+                    fullScreenItem.timerSwipe.stop()
+                    
+                    // TODO: Disabled word selection until input_method.hasSelection is fixed in QtWebEngine
```

Comment:
Do you have a bug reference for this comment?

Reply:
Here's the bug about it although I was also the one that raised it and it's still tag as "needs confirmation"
https://github.com/ubports/ubuntu-touch/issues/1157

Reply:
Please add these to the comment so future code travelers know if the bug has been fixed.